src/sentry/static/sentry/app/components/contextPickerModal.jsx

```diff
@@ -160,7 +172,28 @@ class ContextPickerModal extends React.Component {
     );
   };
 
-  handleSelectOrganization = value => {
+  focusProjectSelector = () => {
+    if (!this.projectSelect || this.state.loading) return;
+
+    // eslint-disable-next-line react/no-find-dom-node
+    ReactDOM.findDOMNode(this.projectSelect)
```

Comment:
We should be able to use forwardRef to clean this up, but there's some issues with `react-test-renderer` serializing it.